src/oscar/apps/checkout/views.py

```diff
@@ -327,7 +330,7 @@ def form_invalid(self, form):
         return super().form_invalid(form)
 
     def get_success_response(self):
-        return redirect('checkout:payment-method')
+        return redirect(self.success_url)
```

Comment:
To be consistent with how `FormView` operates I think this should be:

    return redirect(self.get_success_url())